Make it more clear that there are assets are under a creative commons license

flo opened the pull request:
### Contains

* A hint addition to the LICENSE file to make it clear that some assets are under a creative commons license. 
* A more specific explanation which files are under that creative commons license.
** Some of the files @rzats added were under that license, I listed those files more explicitly so that we later know what files exactly are under that license.
** Some of the icons were made by me, I added now explictly that those are also under the same create commons license like the other icons.
5"

### Outstanding before merging

@rzats please review the icon list if I got it correctly.

GooeyHub commented:
Hooray Jenkins reported success with all tests good!

rzats commented on docs/Credits.md:
```diff
@@ -29,11 +29,56 @@ Soundtrack and Sound Effects
+* editor_zoomablelayout.png
+
+
+Some icons have been made by Florian K�berle and are licensed under the [Creative Commons Attribution 3.0 License](https://creativecommons.org/licenses/by/3.0/):
```
The second character in your surname (assuming that's ö) is replaced by � (desktop Chrome). Might be an encoding issue?

Cervator replied:
Same here (Firefox on Win10)

rzats commented on LICENSE:
```diff
@@ -1,3 +1,10 @@
+The source code is licenced under Appache Licence Version 2.0 (see below).
```
This is a little odd - I've never seen a LICENSE file that includes anything but the license text itself. I think Credits.md is a fairly reasonable location for the attribution notices, though they could be moved to a separate file (e.g. Attribution.md / ThirdPartyNotices.md).

Cervator replied:
Yeah we may want to keep LICENSE unmodified, include the CC license in a different file, and describe the details in Credits.md (or a new file). I just noticed yesterday that GitHub now even auto-detects the license and shows it in the little repo details bar, maybe a modified file would cause trouble there?

flo replied:
If I hadn't known about the addition of the nui editor icons I would have assumed that everything is under the Apache license. The credits file is not that obvious to find, as it is in a sub folder.

Maybe we should create a file called ASSET_LICENSES that contains the reference to the credits file, pretty much like the section I added to the LICENSE file.

rzats commented on docs/Credits.md:
```diff
@@ -29,11 +29,56 @@ Soundtrack and Sound Effects
+* contractOver.png
+* expand.png
+* expandOver.png
+
```
Icon list looks good!

Cervator replied:
Is the 3.0 on purpose? 4.0 is the latest CC at this point.

flo replied:
It's the same license the other icons used.

rzats commented on docs/Credits.md:
```diff
@@ -29,11 +29,56 @@ Soundtrack and Sound Effects
+
+
+
+
```
Some extra newlines here - should probably be limited to one?

flo replied:
nice find. I fixed it

Cervator commented:
Thanks for kicking this off @flo - much needed :-)

As mentioned above I think we should include the full license text for CC BY 4.0 in another LICENSE style file then describe the details elsewhere.

GooeyHub commented:
Hooray Jenkins reported success with all tests good!

flo commented:
I updated the PR again. The LICENCE file is now untouched. Instead i added LICENCE_NOTES files to the directories with the assets.

So this PR should now prevent someone from wrongly assuming that the icons are under the same Apache license like the source code.

I also put now my icons under CC BY 4.0 as suggested bz @Cervator .

The PR contains also a fix the extra whitespaces found by @rzats  and a encoding fix.

GooeyHub commented:
Hooray Jenkins reported success with all tests good!

GooeyHub commented:
Hooray Jenkins reported success with all tests good!

Cervator commented:
Thanks again @flo :)

Went ahead and merged this then added some clarifications to the game readme as well. Noted that anything artwork related should be considered under CC BY 4.0 unless indicated otherwise. As such I think we could eventually leave the list in the Credits file as *exceptions only* - but right now we aren't really ready to consider that in full effect (we still have a few assets to replace and clarify - like with #2407)

I also added a `LICENSE_ARTWORK` file with the full CC BY 4.0 text. Can't actually find that stated as a requirement but I figure it could help clarify (avoids a possible perception of everything being under Apache 2.0 since there's only `LICENSE`)

Also added a second license badge thingie for completeness :-)

See commit https://github.com/MovingBlocks/Terasology/commit/4e0c6b0d9b0b77e4572de3decc18338bbfc2b48f for details and let me know if anything sticks out.

flo commented:
@cervator As far as I understand it you don't need to include a lucense file as long as you reference it. It seems however to require you to mention the origin (author) of the files with a clear statment what got modified. So I think we schould replace LICENSE_ARTWORK with a hint (like the files I added) that artworks might have different licenses that are explained in the credits file. It could of course also mentuon what license new artwork ideally should have. Although when reusing existing artwork the license is fixed. So maybe we want to state that certain artwirk licenses are fine as long they are clearly stated.

Cervator commented:
Yeah I think we need an overall better attribution system for artwork entirely. Like the GDoc I put together while researching block textures, or something like that.

Definitely more work to do, but we'll take it one step at a time I figure :-)
